modules/instrument_builder/jsx/react.questions.js

```diff
@@ -631,6 +631,15 @@ var AddElement = React.createClass({
         });
         hasError = true;
       }
+      if (minYear > 9999 || minYear < 1900 || maxYear > 9999 || maxYear < 1900) {
```

Comment:
What happen if the min or max year entered is 1818 (between 1000 and 1900)? It as 4 digit but would still fail with a message saying it need 4 digit.

@driusan @ridz1208 
Would it be better to have a different message stating that year under 1900 are invalid?

Reply:
Why don't we want to be supporting years < 1900? There are questions someone might want to ask where that would be a valid answer.

Reply:
I don't expect people being 200 years old but should it be a different error message than date must be 4 chars long?  1818 is 4 chars long after all.

Reply:
Years don't need to be ages. There are perfectly reasonable question to ask about years in a study that aren't directly related to the person's age ("When was the house that you were born in built?", "What year did your family migrate to Nunavut?", "When were your parents born?" (in a study relating to the elderly) etc)

Reply:
either change code check for `... minYear < 1900 ...` to `... minYear < 1000 ...` or text of the error message.

Reply:
I agree with dave, making the changes now